sdk/src/main/java/com/horizen/account/transaction/EthereumTransaction.java

```diff
@@ -236,7 +236,8 @@ public void semanticValidity() throws TransactionSemanticValidityException {
             throw new TransactionSemanticValidityException(String.format("Transaction [%s] is semantically invalid: " +
                     "non-positive gas limit", id()));
         if (!BigIntegerUtil.isUint64(getGasLimit()))
-            throw new GasUintOverflowException();
```

Comment:
i'm curios why before the exception type was `GasUintOverflowException`?
Has it any impact on state validation? For example, it led to different result in terms of gas consumption in case of such an error?

Reply:
No, I just changed the exception type for having uniform error handling in this method